front-end/src/components/Post/Poll/GeneralSignals.tsx

```diff
@@ -92,16 +93,40 @@ const CouncilSignals = ({ ayes, className, endBlock, nays, ownVote, pollId, refe
 					onClickAye={() => castVote(Vote.AYE)}
 					onClickNay={() => castVote(Vote.NAY)}
 				/>
-				{!canVote && <span>Poll finished</span>}
-				{ownVote && canVote &&
-					<ButtonLink className='info text-muted' onClick={cancelVote}>Cancel {ownVote.toLowerCase()} vote</ButtonLink>
-				}
+				<div className='cancelEnd'>
+					{ownVote && canVote &&
+						<>
+							<ButtonLink className='info text-muted cancelVoteLink' onClick={cancelVote}>
+								Cancel {ownVote.toLowerCase()} vote
+							</ButtonLink>
+							•
+						</>
+					}
+					{canVote
+						? <span className='pollEndTime'>Poll ends in <BlockCountdown endBlock={endBlock}/></span>
+						: <span>Poll ended.</span>
+					}
+				</div>
 			</Form>
 		</Card>
 	);
 };
 
 export default styled(CouncilSignals)`
+	.cancelEnd{
+		.cancelVoteLink {
+			margin-right: 1rem;
+		}
+		.pollEndTime {
+			margin-left: 1rem;
```

Comment:
This margin makes things not aligned when there's no cancel vote button. Not nice markup-wise, but maybe add a `span` to the bullet point with class `pollEndTime`.

Reply:
thanks, I addressed that.
![image](https://user-images.githubusercontent.com/33178835/86765795-f132b800-c049-11ea-8288-44728cfcdcf4.png)